libobs/obs-properties.c

```diff
@@ -1119,7 +1119,7 @@ void obs_property_button_set_url(obs_property_t *p, char *url)
 	if (!data)
 		return;
 
-	data->url = url;
+	data->url = bstrdup(url);
```

Comment:
This value should be freed.

Reply:
Sorry I overlooked that. Added a cleanup function.

Reply:
Rebased and squashed commits.